crates/apub/src/activities/mod.rs

```diff
@@ -63,11 +64,8 @@ async fn verify_person(
   context: &Data<LemmyContext>,
 ) -> LemmyResult<()> {
   let person = person_id.dereference(context).await?;
-  if person.banned {
-    Err(FederationError::PersonIsBannedFromSite(person.ap_id.to_string()).into())
-  } else {
-    Ok(())
-  }
+  InstanceActions::check_ban(&mut context.pool(), person.id, person.instance_id).await?;
```

Comment:
This should probably be checking *both* a local and remote home ban.

Reply:
If the user was banned by the home instance he cant send us activities anyway (unless he was unbanned and that action didnt federate). Though its right that this should pass the local instance id.